pages/resources/[page].js

```diff
@@ -20,35 +31,98 @@ ResourcesPage.propTypes = {
       downvotes: PropTypes.number,
     }),
   ).isRequired,
-  totalPages: PropTypes.number.isRequired,
+  // eslint-disable-next-line react/no-unused-prop-types
+  defaultTotalPages: PropTypes.number.isRequired,
 };
 
-ResourcesPage.getInitialProps = async ({ pathname, query }) => {
-  const { page = 1 } = query;
-
-  const response = await getResourcesPromise({ page });
+ResourcesPage.defaultProps = {
+  currentPage: 1,
+};
 
+ResourcesPage.getInitialProps = async ({ pathname, query }) => {
```

Comment:
I was recommending that we get away from `getInitialProps` and just do everything on the client! You can still get `pathname` and `query` from `useRouter`